low-level/sensors/impl/centos_7/service_monitor.py

```diff
@@ -461,6 +499,12 @@ def raise_iem(self, service, alert_type):
 
     @classmethod
     def get_alert(cls, service, alert):
+        if service.state == "active":
+            description = alert.description.format(
+                service.name, service.state, service.active_threshold)
+        else:
+            description = alert.description.format(
+                service.name, service.state, service.nonactive_threshold)
```

Comment:
Hi, what happens in case of `FailedAlert`? it has only two format args to replace.
`"{} is in {} state"`